include/echoFunctions.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+
+function echoHeader($Title, $PageName) {
+
+	?>
+	<script src="/include/jquery.js"></script>
+	<?php
+
+	echo "
+		<head>
+			<title>$Title</title>
+			<link rel='stylesheet' href='style.css'/>
+			<link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Quicksand'>
+			<link href='https://fonts.googleapis.com/css?family=Oxygen' rel='stylesheet'>
+		</head>
+
+";
+?>
+
+<script type='text/javascript'>
+
+
+function intakeVote(eventID, userID, direction){
+
+//both buttons haven't been pressed
+	if(!document.getElementById("upVoteButton_"+eventID).classList.contains('voted') && !document.getElementById("downVoteButton_"+eventID).classList.contains('voted')){
+		if(direction == "up"){
+			jsUpVote(eventID, userID, direction);
+		}
+		else{ //direction is down
+			jsDownVote(eventID, userID, direction);
+		}
+	}
+
+//if vote exists
+
+	//the downvote has been pressed but not the upvote
+	else if(!document.getElementById("upVoteButton_"+eventID).classList.contains('voted') && document.getElementById("downVoteButton_"+eventID).classList.contains('voted')){
+		if(direction == "up"){
+			undoDownVote(eventID, userID, direction, "double");
+			jsUpVote(eventID, userID, direction);
+		}
+		else{ //direction is down
+			undoDownVote(eventID, userID, direction, null);
+		}
+	}
+
+	//the upvote has been pressed but not the downvote
+	else{
+		if(direction == "up"){
+			undoUpVote(eventID, userID, direction, null);
+		}
+		else{ //direction is down
+			undoUpVote(eventID, userID, direction, "double");
+			jsDownVote(eventID, userID, direction);
+		}
+
+	}
+}
+
+
+
+function jsUpVote(eventID, userID, direction){
+	//update UI so gets unvoted,
+	//ajax request to unvote this in the server.
+	document.getElementById("upVoteButton_"+eventID).classList.add('voted');
+
+	//gets the number
+	var insideTheDiv = document.getElementById("eventWrapper_"+eventID).innerHTML;
+	var pureNumber = parseInt(insideTheDiv);
+	console.log("before you voted = " +pureNumber);
+
+	//changes the number in the UI
+	document.getElementById("eventWrapper_"+eventID).innerHTML = pureNumber+1;
+	console.log("after you voted = " +(pureNumber+1));
+	// console.log(eventID, userID, direction);
+
+	//call ajax
+	$.post({ url:'/ajax/ajaxVote.php', data:{eventID, userID, direction}});
+
+}
+
+
+function jsDownVote(eventID, userID, direction){
```

Comment:
It you're looking for a challenge, there's a way to abstract this stuff a level higher so that upvoting and downvoting use the exact same functions. Like, imagine having a "vote" function that takes a direction. You could, or example, get the current vote for that event, and see "is it the same as the direction of the vote" (like, they could both be "up" or one could be "up" and one could be "down" etc.). This isn't strictly necessary, your current code seems abstract enough, but that would remove the need for a lot of this code that is very similar but not quite duplicated.